pandas/tests/io/test_sql.py

```diff
@@ -2013,6 +2013,15 @@ def test_to_sql_with_negative_npinf(self, input):
         df = DataFrame(input)
 
         if self.flavor == "mysql":
+            # GH 36465
+            # The input {"foo": [-np.inf], "infe0": ["bar"]} does not raise any error
+            # for pymysql version >= 0.10
+            # TODO: remove this version check after GH 36465 is fixed
+            import pymysql
+
+            if pymysql.VERSION[0:3] >= (0, 10, 0) and "infe0" in df.columns:
+                return
```

Comment:
actually can you make this xfail like this: https://github.com/pandas-dev/pandas/blob/master/pandas/tests/indexing/test_coercion.py#L116